### ISSUE TITLE
URL in Readme broken

### ISSUE THREAD

**Issue by juancarlospaco** [ardek66/geminim#3]
Demo URL in Readme https://gemi.nim.town/index.gemini is unreachable as of today. 🙂

**Comment by ardek66** [comment]
Yeah I still haven't got a new server to host it, I'll get a VPS these days for it, thanks for the heads up

**Comment by ardek66** [comment]
I got a VPS and setup a temporary Gemini page, I will add the HTML pages when I update the content a bit more.

gemini://gemi.nim.town

**Comment by luntik2012** [comment]
Just a necro-bump, `gemi.nim.town` dns record does not exist

Could you add some examples/tutorial into README?

**Comment by ardek66** [comment]
> Could you add some examples/tutorial into README?

the example files (config.ini) already contain all the options with comments
